app/src/main/java/au/com/bluedot/minimalintegration/MainApplication.java

```diff
@@ -249,16 +155,24 @@ private Notification createNotification() {
             return notification.build();
         } else {
 
-            NotificationCompat.Builder notification = new NotificationCompat.Builder(getApplicationContext())
+            NotificationCompat.Builder notification = new NotificationCompat.Builder(getApplicationContext(), channelId)
                     .setContentTitle(getString(R.string.foreground_notification_title))
                     .setContentText(getString(R.string.foreground_notification_text))
                     .setStyle(new NotificationCompat.BigTextStyle().bigText(getString(R.string.foreground_notification_text)))
                     .setOngoing(true)
-                    .setCategory(Notification.CATEGORY_SERVICE)
                     .setPriority(PRIORITY_MAX)
                     .setSmallIcon(R.mipmap.ic_launcher);
-
             return notification.build();
         }
     }
+
+    @Override public void onTempoResult(@Nullable BDError bdError) {
+        String text = "Tempo start";
```

Comment:
Not sure the string concatenation is really saving us much here, but we should at least put a space between start and the next word.

Reply:
fixed it